templates/kialioperator-vpa.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" }}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: kiali-kiali-operator
```

Comment:
Maybe `kiali-operator` instead of `kiali-kiali-operator`?
Maybe use anchor?

Reply:
good catch, it's much better to use an anchor here.